inc/template.php

```diff
@@ -857,10 +855,10 @@ function tpl_youarehere($sep = null, $return = false)
     $parts = explode(':', $ID);
     $count = count($parts);
 
-    $out .= '<span class="bchead">' . $lang['youarehere'] . ' </span>';
+    $out .= '<h2 class="bchead">' . $lang['youarehere'] . ' </h2>';
 
     // always print the startpage
-    $out .= '<span class="home">' . tpl_pagelink(':' . $conf['start'], null, true) . '</span>';
+    $out .= '<ol><li class="home">' . tpl_pagelink(':' . $conf['start'], null, true) . '</li>';
```

Comment:
Fully agree. I have also added some `DOKU_LF`s and `DOKU_TAB`s to removed some spaces, to make the generated HTML code a bit neater. I didn't add the proper indentation to fit the code around it, as this would also need a larger cleanup of the template. I have an approach in my own (Ad Hominem) template, though, to ensure the HTML is properly indented (as good as it goes anyways :-)

Reply:
we used to add newlines and spaces to generated HTML but no longer do. with modern code inspection built into browsers the raw html source no longer matters and leaving out such cosmetics makes the source easier to read and maintain.

Reply:
Ah, OK, removed them again (except for the one after the H2, as this makes it easier to find the list in the source code).

This is where I generally have a different approach, as I find that I will often read HTML source code, and the more readable that is, the better time I have with it. And even though I normally try to optimize for file sizes - this is an exception, as this really effects my debugging time in the HTML.

Nevertheless, I'm not using the default template, so I am happy to follow the established style :-)

But maybe we could consider some kind of "neatnik-mode" in the core HTML renderer which adds tabs (coming to think of it, maybe the "debug" mode could do that... ;-)